### PR TITLE
add tariffID in OCPI settings

### PR DESCRIPTION
**Pull request by rmangin-fr**

### CODE REVIEW

File: src/server/rest/v1/service/SettingService.ts [hunk 1]
```diff
@@ -176,67 +177,97 @@ export default class SettingService {
...
+        filteredRequest = SettingValidator.getInstance().validateSettingSmartChargingSetReq(req.body);
+        break;
+      case IntegrationSettings.REFUND:
+        filteredRequest = SettingValidator.getInstance().validateSettingRefundSetReq(req.body);
+        break;
+      case IntegrationSettings.PRICING:
+        filteredRequest = SettingValidator.getInstance().validateSettingPricingSetReq(req.body);
+        break;
+      default:
+        filteredRequest = settingUpdate;
```
[the remark this answers is not in the document]
Comment (rmangin-fr) [reply src/server/rest/v1/service/SettingService.ts]: done :)

File: src/server/rest/v1/service/SettingService.ts [hunk 2]
```diff
@@ -277,25 +307,25 @@ export default class SettingService {
         if (Utils.hash(settingUpdate.content.crypto.key) !== Utils.hash(setting.content.crypto.key)) {
```
Comment (ClaudeROSSI) [inline src/server/rest/v1/service/SettingService.ts:307]: settingUpdate ... 
does not compile!

**Review by ClaudeROSSI — CHANGES_REQUESTED** [review]
does not compile anymore.

check my comments

**Review by ClaudeROSSI — APPROVED** [review]

**Comment by LucasBrazi06** [comment]
@rmangin-fr Conflicting files.

**Review by LucasBrazi06 — APPROVED** [review]

**Pull request merged**